setup.py

```diff
@@ -78,6 +78,14 @@
 package_info['package_data'].setdefault(PACKAGENAME, [])
 package_info['package_data'][PACKAGENAME].append('data/*')
 
+# Adam's note: I don't know what to do with this.
+# # Define entry points for command-line scripts
+# entry_points = {}
+# entry_points['console_scripts'] = [
+#     'astropy-package-template-example = packagename.example_mod:main',
+# ]
```

Comment:
@cdeil since you were helping with package-template-merging stuff, do you have any idea what I should be doing with this segment?